universal_transfer_operator/docs/configurations.rst

```diff
@@ -0,0 +1,49 @@
+Configuration
+=============
+
+Configuring the database default schema
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Universal Transfer Operator uses the global default schema configuration for the ``Table`` metadata, unless users
+specify a different specific ``Table`` (metadata) ``schema``.
+
+User can define the default schema in two ways:
+1. Globally, for all the databases
+2. Individually, for each database
+
+The Universal transfer operator applies the global default schema (``tmp_transfers`` by default) when the user does not
+set up a database-specific configuration. Example:
+environment variable :
+
+.. code:: ini
+
+   AIRFLOW__UNIVERSAL_TRANSFER_OPERATOR__SQL_SCHEMA="tmp_transfers"
```

Comment:
Not a blocker - But in a separate PR, we should verify this behavior, I think we missing test cases for this.

Reply:
Added the issue to add test cases: https://github.com/astronomer/astro-sdk/issues/1864